osu.Game/Skinning/LegacyJudgementPieceOld.cs

```diff
@@ -55,6 +56,9 @@ public virtual void PlayAnimation()
                     this.ScaleTo(1.6f);
                     this.ScaleTo(1, 100, Easing.In);
 
+                    this.MoveTo(new Vector2(0, -5));
+                    this.MoveToOffset(new Vector2(0, 80), fade_out_delay + fade_out_length, Easing.In);
```

Comment:
Looking at stable source, I doubt this should be here (see `SkinManager.UseNewLayout` check mentioned in review comment above). `LegacyJudgementPieceOld` is for version 1.0 skins. Did you test this PR with one? Are you sure this PR also matches behaviour for such skins?